recipes/user.rb

```diff
@@ -13,6 +13,7 @@
 
 user conf['user'] do
   home conf.home
+  group conf.group
```

Comment:
Also can you change the above lines to `conf['home']` and `conf['group']` notation?

I got warnings that it will be deprecated:
> Method access to node attributes (node.foo.bar) is deprecated and will be removed in Chef 13, please use bracket syntax (node["foo"]["bar"])